web/about/views.py

```diff
@@ -38,8 +38,12 @@ def index(request):
         'next_event': events.get_next(),
         'has_registered': False,
         'chat_form': None,
-        'num_members': len(members()),
-        'num_active': len(active_members()),
+        # Prod with access to Slack API Token
+        # 'num_members': len(members()),
+        # 'num_active': len(active_members()),
+        # Dev Mode with no access to Slack API Token
+        'num_members': 900,
+        'num_active': 0,
```

Comment:
We'll need to put this back before merging. I'll see about making a patch you can integrate that handles the case of not having a valid token.

Reply:
Got it. 
I'll adjust or remove that piece.

Reply:
I commented out the other way. 
That way it's still there if someone needs it but can easily be removed.